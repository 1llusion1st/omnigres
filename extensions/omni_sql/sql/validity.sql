-- Valid
SELECT omni_sql.is_valid('SELECT'::omni_sql.statement);

-- An invalid table
SELECT omni_sql.is_valid('SELECT * FROM no_such_table_exists'::omni_sql.statement);

-- Parameters
SELECT omni_sql.is_valid('SELECT $1'::omni_sql.statement);

-- Multiple statements (one is invalid)
SELECT omni_sql.is_valid('SELECT; SELECT $1'::omni_sql.statement);
