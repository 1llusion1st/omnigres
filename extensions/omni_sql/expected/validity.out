-- Valid
SELECT omni_sql.is_valid('SELECT'::omni_sql.statement);
 is_valid 
----------
 t
(1 row)

-- An invalid table
SELECT omni_sql.is_valid('SELECT * FROM no_such_table_exists'::omni_sql.statement);
 is_valid 
----------
 f
(1 row)

-- Parameters
SELECT omni_sql.is_valid('SELECT $1'::omni_sql.statement);
 is_valid 
----------
 f
(1 row)

-- Multiple statements (one is invalid)
SELECT omni_sql.is_valid('SELECT; SELECT $1'::omni_sql.statement);
 is_valid 
----------
 f
(1 row)

